README.md

```diff
@@ -1,59 +1,94 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Shows
 
-> One paragraph statement about the project.
+> In this project, we build a site using the TVmazAPI. It displays information about tv shows and we also used an involvement api to interact and update every movie. The interactions we have implemented are giving likes and comments. After every interaction the page is updated.
+
+> Here is a little preview of the app with the creators [preview](https://drive.google.com/file/d/1eN6UrF6mOuyHjrpnmfRoFWFGzjdlGSTP/view?usp=sharing).
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML&CSS, JavaScript
+- Webpack, Jest, Involment API, TVMaze API
+- Kanban
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://wes-isaac.github.io/Capstone_project_2/)
```

Comment:
- [ ] As it requirement for this project, please kindly add a video presentation of your project and add the link here. Please check the screenshot below and make appropriate changes
![wes](https://user-images.githubusercontent.com/65852621/142597369-e615755e-7aff-48e8-9035-1a334189d124.png)
.

Reply:
Hi, @Mihndim2020,
Thanks for the review. we have a video in our readme file after project description maybe it not visible we will move to live demo section, thanks!!